sftp-source/src/test/resources/logback.xml

```diff
@@ -0,0 +1,11 @@
+<configuration>
```

Comment:
Again; I started with the existing FTP source, which has one of these.

I'll remove both.